### PR TITLE
Content amends for seo

### PR DESCRIPTION
**Pull request by rodmiromind**

### CODE REVIEW

File: content/en/docs/installing-on-linux/introduction/multioverview.md [hunk 1]
```diff
@@ -172,13 +172,19 @@ Here are some examples for your reference:
...
 A default file `config-sample.yaml` will be created if you do not change the name. Edit the file and here is an example of the configuration file of a multi-node cluster with one master node.
 
 {{< notice note >}}
 
-To customize Kubernetes related parameters, refer to [Kubernetes Cluster Configurations](../vars/).
+To customize 
+
+
+
+
```
Comment (FeynmanZhou) [inline content/en/docs/installing-on-linux/introduction/multioverview.md:185]: There are a few redundant blank lines here, please delete them.

File: content/en/blogs/understand-requests-and-limits-in-kubernetes.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Dive Deep into Resource Requests and Limits in Kubernetes'
+title: 'Kubernetes Resource Requests and Limits'
```
Comment (FeynmanZhou) [inline content/en/blogs/understand-requests-and-limits-in-kubernetes.md:2]: How about `Kubernetes Resource Requests and Limits Deep Dive`?

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
@rodmiromind Please check out my comments. Thanks!

File: content/en/docs/installing-on-linux/introduction/multioverview.md (hunk 1 above)
Comment (rodmiromind) [reply content/en/docs/installing-on-linux/introduction/multioverview.md]: Done

File: content/en/blogs/understand-requests-and-limits-in-kubernetes.md (hunk 2 above)
Comment (rodmiromind) [reply content/en/blogs/understand-requests-and-limits-in-kubernetes.md:2]: Sounds great

**Pull request closed without merging**